src/Stratis.Bitcoin.Features.Miner/PowBlockDefinition.cs

```diff
@@ -76,11 +76,12 @@ public void TestBlockValidity()
         {
             this.logger.LogTrace("()");
 
-            RuleContext context = this.consensusRules.CreateRuleContext(new ValidationContext { BlockToValidate = this.block, ChainedHeaderToValidate = this.ConsensusManager.Tip });
-            context.MinedBlock = true;
-
-            // TODO: Is this correct or should we not call validation from rules but from CM
-            this.ConsensusManager.ConsensusRules.PartialValidationAsync(new ValidationContext { BlockToValidate = this.block, ChainedHeaderToValidate = this.ConsensusManager.Tip });
+            // TODO ACTIVATION pass header of a block that is being validated, not CT!
```

Comment:
?

Reply:
Because of the wrong naming of the variable someone passed there CT so now we have wrong logic here. 
Commented it out and said how to fix it.